src/main/kotlin/org/dashj/platform/dpp/errors/concensus/ConcensusException.kt

```diff
@@ -76,80 +76,81 @@ abstract class ConcensusException(message: String) : DPPException(message) {
+////                Codes.InvalidIdentityAssetLockTransactionOutputError -> InvalidIdentityAssetLockTransactionOutputException(arguments)
+////                Codes.InvalidIdentityPublicKeyDataError -> InvalidIdentityPublicKeyDataException(arguments)
+//                Codes.InvalidInstantAssetLockProofError -> InvalidInstantAssetLockProofException(arguments)
+//                Codes.InvalidInstantAssetLockProofSignatureError ->
+//                    InvalidInstantAssetLockProofSignatureException(arguments)
+//                Codes.InvalidStateTransitionTypeError -> InvalidStateTransitionTypeException(arguments)
+//                Codes.MissingStateTransitionTypeError -> MissingStateTransitionTypeException()
+////                Codes.StateTransitionMaxSizeExceededError -> StateTransitionMaxSizeExceededException(arguments)
+//                Codes.IdentityNotFoundError -> IdentityNotFoundException(arguments)
+//                Codes.InvalidIdentityPublicKeyTypeError -> InvalidIdentityPublicKeyTypeException(arguments)
+//                Codes.InvalidStateTransitionSignatureError -> InvalidStateTransitionSignatureException()
+//                Codes.MissingPublicKeyError -> MissingPublicKeyException(arguments)
+//                Codes.PublicKeyIsDisabledError -> PublicKeyIsDisabledException(arguments)
+//                Codes.InvalidSignaturePublicKeySecurityLevelError ->
+//                    InvalidSignaturePublicKeySecurityLevelException(arguments)
+//                Codes.BalanceIsNotEnoughError -> BalanceIsNotEnoughException(arguments)
+////                Codes.DataContractAlreadyPresentError -> DataContractAlreadyPresentException(arguments)
+////                Codes.DataTriggerConditionError -> DataTriggerConditionException(arguments)
+////                Codes.DataTriggerExecutionError -> DataTriggerExecutionException(arguments)
+////                Codes.DataTriggerInvalidResultError -> DataTriggerInvalidResultException(arguments)
+//                Codes.DocumentAlreadyPresentError -> DocumentAlreadyPresentException(arguments)
+////                Codes.DocumentNotFoundError -> DocumentNotFoundException(arguments)
+////                Codes.DocumentOwnerIdMismatchError -> DocumentOwnerIdMismatchException(arguments)
+////                Codes.DocumentTimestampsMismatchError -> DocumentTimestampsMismatchException(arguments)
+////                Codes.DocumentTimestampWindowViolationError -> DocumentTimestampWindowViolationException(arguments)
+////                Codes.DuplicateUniqueIndexError -> DuplicateUniqueIndexException(arguments)
+//                Codes.InvalidDocumentRevisionError -> InvalidDocumentRevisionException(arguments)
+//                Codes.IdentityAlreadyExistsError -> IdentityAlreadyExistsException(arguments)
+//                else -> UnknownConcensusError(code.code, arguments)
+//            }
```

Comment:
Is this all broken now?

Reply:
Yes, changes in Platform 0.24 resulted in a new serialization format for error codes.